src/main/groovy/edu/oregonstate/mist/api/Resource.groovy

```diff
@@ -9,8 +9,12 @@ import javax.ws.rs.core.Response.ResponseBuilder
 abstract class Resource {
     protected static Properties properties = new Properties()
 
-    public static loadProperties(String fileName) {
-        properties.load(new FileReader(fileName))
+    public static loadProperties() {
+        def stream = this.getResourceAsStream('resource.properties')
+        if (stream == null) {
+            throw new Exception("couldn't open resource.properties")
+        }
+        properties.load(stream)
```

Comment:
This logic is a bit repetitive with the InfoResource. The InfoResource inherits this class and could just reuse the logic.